app/src/main/res/layout/fragment_about.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.AboutFragment">
+
+    <TextView
+        android:id="@+id/appName"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:text="Wallet Guru"
+        android:textSize="30sp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/appSlogan"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:text="Guiding you to stay on top of your finances."
+        android:textSize="25sp"
+
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/appName" />
+
+    <TextView
+        android:id="@+id/appAbout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:textStyle="italic"
+        android:text="Wallet Guru is a personalized financial information platform based on providing user-relevant Reddit posts from logged expense information. Users can track balances over extended periods using different categories (ex. Investments, savings, debt balances, personal budgets) and be suggested posts in order of importance based on their performance. Using our simple logging system, track all your expenses in seconds – simply input information for each category to see your article feed instantly update to accommodate the changes in performance. Our proprietary sorting algorithm will ensure you can swipe away posts you’ve already read and stay up to date every time you open the app. "
```

Comment:
For small strings like titles and such I don't care, but this kind of string is very ugly in code. I think we should extract it as a string resource (which has the added benefit of making it translatable).

Reply:
extracted as string resource.